### ISSUE TITLE
NPM START ERROR

### ISSUE THREAD

**Issue by nevets2k17** [npm/npm#19062]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [x] npm is doing something I don't understand.
- [ ] npm is producing an incorrect install.
    - [ ] Other (_see below for feature requests_):

**pi@raspberrypi:~/MagicMirror $ npm start

> magicmirror@2.1.3 start /home/pi/MagicMirror
> sh run-start.sh

run-start.sh: 4: run-start.sh: electron: not found

npm ERR! Linux 4.9.59-v7+
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "start"
npm ERR! node v6.11.5
npm ERR! npm  v3.10.10
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! magicmirror@2.1.3 start: `sh run-start.sh`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the magicmirror@2.1.3 start script 'sh run-start.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the magicmirror package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     sh run-start.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs magicmirror
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls magicmirror
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /home/pi/MagicMirror/npm-debug.log
pi@raspberrypi:~/MagicMirror $ 
**

enyone can help me ??

**Comment by kenany** [comment]
Did you run `npm install` first?

**Comment by nevets2k17** [comment]
Yes sir but once npm install is on 90% of installing my raspbpi is freezing how to fix this sir :(
